src/scss/layout/_footer.scss

```diff
@@ -0,0 +1,48 @@
+.footer {
+  background: #000;
+  padding: 59px 91px 64px 30px;
+
+&__logo {
+  width: 117px;
+  height: 48px;
+  margin-bottom: 40px;
```

Comment:
they are not in alphabetical order

Reply:
I already organized logo style attributes alphabetically

Reply:
I do not see the alphabetical order, please commit code.

Reply:
I already organized style attributes alphabetically